handlers/handlers.go

```diff
@@ -56,6 +60,55 @@ func previousReleasesSample(w http.ResponseWriter, req *http.Request, rc RenderC
 	rc.BuildPage(w, m, "previousreleases")
 }
 
+func ReleaseCalendar(cfg config.Config, rc RenderClient, api SearchAPI) http.HandlerFunc {
+	return dphandlers.ControllerHandler(func(w http.ResponseWriter, r *http.Request, lang, collectionID, accessToken string) {
+		releaseCalendar(w, r, accessToken, collectionID, lang, rc, api, cfg)
+	})
+}
+
```

Comment:
can we validate sort, keywords, etc here as well ?

Reply:
Yes we can. I was only doing the 'user settable' ones here as the dp-search-api does the complete validation as well. But I wasn't sure if 'keywords' had a validation, as I have been treating it as an opaque string which is passed directly to Elastic Search. Do you happen to know what validation I should be doing with it?

Reply:
So I think the API should validate all user passed parameters to ensure they are set to expected values